src/features/deposit/services/depositService.ts

```diff
@@ -181,6 +183,12 @@ export class DepositService {
       contractId,
       accountId,
     })
-    return storageBalance < BigInt(FT_MINIMUM_STORAGE_BALANCE)
+    return storageBalance < (await this.getMinimumStorageBalance(contractId))
```

Comment:
So you will do 2 requests for the same data, right?

Reply:
I see the confusion. No, we actually make two different requests. The first is `getNearNep141StorageBalance`, which uses the method `storage_balance_of` to tell us the user's deposit. The second is `getNearNep141MinStorageBalance`, which uses the method `storage_balance_bounds` to tell us the minimum required deposit value.